addons/project/tests/test_project_flow.py

```diff
@@ -34,7 +34,7 @@
 Integrator at Agrolait"""
 
 
-class TestProjectFlow(TestProjectCommon, MockEmail):
+class TestProjectFlow(TestProjectCommon, MailCommon, MockEmail):
```

Comment:
Since `MailCommon` contained `MockEmail` in its parent class, we could remove, `MockEmail` here, couldn't we? 🤔